aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -1382,6 +1382,60 @@ module aptos_token::token {
         token_data.royalty
     }
 
+    public fun get_tokendata_id(token_id: TokenId): TokenDataId {
```

Comment:
we should decouple this into two PRs or minimally commits.

Reply:
updated to multiple commits